vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/WaitSteps.java

```diff
@@ -370,36 +237,33 @@ public void waitTillFrameAppearsAndSwitchToIt(String frameName)
      * <li><i>Finds</i> an element with the <b>specified name</b> within search context
      * <li><i>Waits</i> until this element becomes stale
      * </ul>
-     * @param elementName Any attribute value or text value of the element
+     * @param locator to locate element
      */
-    @When("I wait until an element with the name '$elementName' is stale")
-    public void waitTillElementIsStale(String elementName)
+    @When("I wait until element located `$locator` is stale")
```

Comment:
line 224 in description:
...with specified <b>name</b> -> ...with specified <b>locator</b>
line 237:
....<b>specified name</b> -> ....<b>specified locator</b>